pyuvdata/uvdata/miriad.py

```diff
@@ -229,7 +229,7 @@ def read_miriad(
                     else:
                         bl_str_list.append(str(bl[1]) + "_" + str(bl[0]))
                         if len(bl) == 3:
-                            bl_pols.add(bl[2][::-1])
+                            bl_pols.add(uvutils.conj_pol(bl[2].lower()))
```

Comment:
This `lower()` is redundant with the `conj_pol` function. And should probably remove to be consistent with the other instances.